onnx/defs/tensor/defs.cc

```diff
@@ -1602,40 +1602,104 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input 'X' and output 'Y' to all tensor types.")
         .SetDoc(Upsample_ver10_doc)
         .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
-          resizeShapeInference(ctx);
+          resizeShapeInference(ctx, false);
         }));
 
-static const char* Resize_ver10_doc = R"DOC(
-Resize the input tensor.
+static const char* Resize_ver11_doc = R"DOC(
+Resize the input tensor. In general, it calculates every value in the output tensor as a weighted average of neighborhood (a.k.a. sampling locations) in the input tensor.
 Each dimension value of the output tensor is:
-  output_dimension = floor(input_dimension * scale).
+  output_dimension = floor(input_dimension * scale) if input \"sizes\" is not specified.
 )DOC";
 
+static const char* Resize_attr_coordinate_transformation_mode_doc = R"DOC(
+This attribute describes how to transform the coordinate in the resized tensor to the coordinate in the original tensor. <br/>
+
+The coordinate of each dimension is transformed individually. Denote x_resized as the coordinate of a specific dimension in the resized tensor, x_original as the coordinate of this dimension in the original tensor, length_original as the length of the original tensor in this dimension, length_resized as the length of the resized tensor in this dimension, roi_x = (start, end) of the this dimension in input "roi", scale = length_resized / length_original, <br/>
+
+if scaler is "half_pixel", <br/>
```

Comment:
```suggestion
if coordinate_transformation_mode is "half_pixel", <br/>
```
and also for the following places.

Reply:
Thanks! I have updated it.